pkg/component/worker/containerd.go

```diff
@@ -91,14 +100,43 @@ func (c *ContainerD) Start(_ context.Context) error {
 
 func (c *ContainerD) setupConfig() error {
 	// If the config file exists, use it as-is
-	if file.Exists(confPath) {
-		return nil
-	}
+	// FIXME We need to figure out how to overwrite this only if it's fully k0s managed
```

Comment:
Do we need it? I think everything in the `/etc/k0s` directory is k0s managed

Reply:
Yes, we need the logic to handle this. All previous versions (before this PR) will treat any existing `/etc/k0s/containerd.toml` file as user managed and we should NOT overwrite those by default.